src/main/msp/msp_serial.c

```diff
@@ -131,6 +133,36 @@ static void mspSerialResponse(mspPort_t *msp, mspPacket_t *reply)
     serialEndWrite(msp->port);
 }
 
+
+static mspPacket_t *mspCreateMessage(void)
```

Comment:
Some packet buffers may be allocated on stack, saving considerable space with current scheduler implementation.

Also this function hides the fact that message is static, possibly leading to confussion (you can't allocate request and reply together)